config/auth.php

```diff
@@ -112,4 +112,5 @@
         ],
     ],
 
+    'iar' => env('APP_REGISTER', false),
```

Comment:
Is there a fixed convention between `PF_`, `APP_` and no prefix in env keys? 

Perhaps it would be worth naming the PHP and env settings consistently, making it easier to map the linage of the various layers of configuration as well

```suggestion
    'in_app_registration' => env('IN_APP_REGISTRATION', false),
```